app/controllers/conferences_controller.rb

```diff
@@ -2,14 +2,12 @@ class ConferencesController < ApplicationController
   before_action :authenticate_user!, except: [:index, :show]
   before_action :confirm_role, except: [:index, :show]
 
+  include Conferencable
```

Comment:
Hm, this name sounds super reasonable if this would be about models... But in controller-context, this I'm afraid does not really name the behavior or "thing" the module is about... 🤔 

Do you think you can come up with a name better describing / "naming" the logic or behavior encapsulated in the new module? Would be ace ✌️ and make it way more understandable for someone looking at these controllers for the first time.